spec/helpers/url_helper.rb

```diff
@@ -15,15 +15,9 @@ def get(path)
   end
 
   def flood_get(path, times:)
-    threads = []
-
     times.times do
-      threads << Thread.new do
-        response = get(path)
-        yield response if block_given?
-      end
+      response = get(path)
+      yield response if block_given?
```

Comment:
Does this still actually test "flood get" if it's not doing it, you know, simultaneously? 😅

Reply:
Ah, let me check, as there’s a lot of flakiness, I removed the threading part to see if this was improved (it seems a little better), I can check again by reverting the code.

Reply:
Doesn’t seem to change a lot of things actually :shrug: